### PR TITLE
refactor(selector-strip): implement and document variables and attributes

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

`<hx-selectorstrip>` : implement and document variables and attributes

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF2126

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 .hxSelector {
   border-radius: 2px;
   color: $cyan-900;
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:6]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@
...
+  color: var(--hxSelector-color, $cyan-900);
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:7]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 3]
```diff
@@ -56,41 +59,53 @@
       // unchecked (focused)
       &:focus + span {
         background-color: $cyan-100;
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:61]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 4]
```diff
@@ -56,41 +59,53 @@
...
+        background-color: var(--hxSelector-unchecked-bgcolor, $cyan-100);
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:62]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 5]
```diff
@@ -56,41 +59,53 @@
...
         color: $cyan-900;
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:63]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 6]
```diff
@@ -56,41 +59,53 @@
...
+        color: var(--hxSelector-unchecked-color, $cyan-900);
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:64]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 7]
```diff
@@ -56,41 +59,53 @@
...
       }
 
       // checked (unfocused)
       &:checked + span {
         background-color: $cyan-900;
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:69]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 8]
```diff
@@ -56,41 +59,53 @@
...
+        background-color: var(--hxSelector-checked-bgcolor, $cyan-900);
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:70]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 9]
```diff
@@ -56,41 +59,53 @@
...
         color: $gray-0;
+        color: var(--hxSelector-checked-color, $gray-0);
       }
 
       // checked (focused)
       &:checked:focus + span {
         box-shadow: 0 0 4px 0 $cyan-900;
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:77]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 10]
```diff
@@ -56,41 +59,53 @@
...
+        box-shadow: 0 0 4px 0 var(--hxSelector-checked-focus-boxShadow, $cyan-900);
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:78]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 11]
```diff
@@ -56,41 +59,53 @@
...
       }
 
       // pressed (checked or unchecked)
       &:active + span {
         background-color: $cyan-700;
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:83]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 12]
```diff
@@ -56,41 +59,53 @@
...
+        background-color: var(--hxSelector-active-bgcolor, $cyan-700);
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:84]: Please use updated  `v2.0` colors to replace `$cyan-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 13]
```diff
@@ -56,41 +59,53 @@
...
+        background-color: var(--hxSelector-invalid-bgcolor, $gray-0);
         border: 2px solid $red-status-500;
+        border: 2px solid var(--hxSelector-invalid-border, $red-status-500);
         color: $red-status-500;
+        color: var(--hxSelector-invalid-color, $red-status-500);
       }
 
       // invalid (checked)
       &[invalid]:checked + span {
         background-color: $red-100;
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:101]: Please use updated  `v2.0` colors to replace `$red-XXX`.

File: src/scss/components/selector-strip/_hxSelector.scss [hunk 14]
```diff
@@ -56,41 +59,53 @@
...
+        background-color: var(--hxSelector-invalid-checked-bgcolor, $red-100);
```
Comment (100stacks) [inline src/scss/components/selector-strip/_hxSelector.scss:102]: Please use updated  `v2.0` colors to replace `$red-XXX`.

**Review by 100stacks — CHANGES_REQUESTED** [review]
### Please use `v2.0` colors variables.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 1 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 2 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 3 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 4 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 5 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 6 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 7 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 8 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 9 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 10 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 11 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 12 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 13 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

File: src/scss/components/selector-strip/_hxSelector.scss (hunk 14 above)
Comment (lalithkarikelli) [reply src/scss/components/selector-strip/_hxSelector.scss]: Changes have been done, please review.

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**